app/controllers/application_controller.rb

```diff
@@ -7,6 +7,7 @@ class ApplicationController < ActionController::Base
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  before_action -> { Current.user = current_user }
```

Comment:
We might want to trigger this form a devise hook rather than assuming it will happen after devise authentication here.